src/mm-route.js

```diff
@@ -83,35 +48,58 @@ angular.module('mmRoute', [
   // Arguments:
   //   routes    {Object}    Map of roles to route definitions
```

Comment:
In fact, there are now numerous problems with the function comments. Can you please go through and ensure each function is correctly documented.